cibuildwheel/linux.py

```diff
@@ -96,12 +96,18 @@ def build(options: BuildOptions):
         ('pp', 'manylinux_x86_64', options.manylinux_images['pypy_x86_64']),
     ]
 
+    if not os.path.realpath(options.package_dir).startswith(os.path.realpath('.')):
```

Comment:
I guess this is strictly speaking only necessary on Linux because of the docker image?
But it does feel like the abstraction gets lost a bit by only enforcing it here, and not in `__main__.py`.

Then again, the old `project_dir` is now only relevant on Linux, I guess? Or does it still have a function on Windows/macOS?

Reply:
'project dir' does still have a function on Windows/macOS, but only as the working dir - it's the cwd when commands like CIBW_BEFORE_BUILD run and CIBW_ENVIRONMENT are executed...

I was wondering about checking this in `__main__.py`, but it just felt a little punitive. It's still documented as 'must be a subpath of cwd', as if it were a global rule. But I don't think it needs to be enforced that strictly.

Reply:
> 'project dir' does still have a function on Windows/macOS, but only as the working dir

But we don't really change the working dir? So somehow, it makes sense that that's just where commands are executed. (But true, it's a good point that these are not executed from `package_dir`, indeed!)

Anyway, I'm happy with this as well. Just a minor remark, when seeing this.